src/main/java/org/mozilla/jss/util/Base64OutputStream.java

```diff
@@ -44,7 +44,8 @@ public Base64OutputStream(PrintStream out, int quadsPerLine) {
 	}
 		
 
-	public void write(int oneByte) throws IOException {
+	@Override
```

Comment:
Not sure if this is already mentioned somewhere else, but it looks like this file uses tabs for indentation. I think generally we want to use 4 spaces for indentation, but this probably can be addressed separately later.

Reply:
I added in an extra commit that does Eclipse auto-formatting to get rid of any tabs.